### ISSUE TITLE
SElinux - improvement. Non sshd specified ports need to be removed

### ISSUE THREAD

**Issue by george-oniceag** [george-oniceag/ansible-sshd-role#6]

**Comment by george-oniceag** [comment]
Task added to role : 
```
# CHECK Selinux status
- name: Get value of selinux status
  shell: getenforce
  register: ck_selinux
  tags: ['sshd-selinux']

# IF selinux is Enforcing or permissive then execute block to correctly configure selinux
- block:
  # GET current status of selinux compliance for SSH
  - name: Get current selinux compliant port for SSHD
    shell: semanage port --list | grep ssh_port_t | awk -F ' ' '{for(i=3;i<=NF;i++)print $i}'
    register: ck_sshd_seport
  # DEFINE working port exclusion fact
  - set_fact:
        sshd_port_exclusion: []
  # POPULATE port exclusion fact with data
  - set_fact:
        sshd_port_exclusion: "{{sshd_port_exclusion}} + ['{{item|regex_replace('(.*),$','\\1')}}']"
    with_items: "{{ck_sshd_seport.stdout_lines}}"
  # REMOVE required active porst from port exclusion fact
  - set_fact:
        sshd_port_exclusion: "{{sshd_port_exclusion|difference(sshd.listen.ports)}}"
  # REMOVE porst specified in the port exclusion list
  - name: Remove unused ports from selinux
    seport: ports={{item}} proto=tcp setype=ssh_port_t state=absent
    with_items: "{{sshd_port_exclusion}}"
  when: ck_selinux.stdout|lower == "enforcing" or ck_selinux.stdout|lower == "permissive"
  tags: ['sshd-selinux']
```

**Comment by george-oniceag** [comment]
BUGfix tests 🆗  
Will be available in the next release v0.2.0
